spikingjelly/activation_based/functional/misc.py

```
@@ -13,9 +16,328 @@
+    layer.BatchNorm2d,
+    layer.BatchNorm3d,
+)
+
+
+def _matches_module_pattern(pattern, node: fx.Node, modules) -> bool:
+    if len(node.args) == 0:
+        return False
+    nodes = (node.args[0], node)
+    if len(pattern) != len(nodes):
+        return False
+    for expected_type, current_node in zip(pattern, nodes):
```

Ruff (0.15.11)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)